project_management/Technical_setup_when_starting_a_new_project.md

```diff
@@ -70,6 +70,16 @@ Setup automerge github action
   - create github automerge action
     - copy auto-merge.yml, dependabot.yml, workflows/auto-merge-dependabot.yml from [muffi/.github](https://github.com/abtion/muffi/tree/main/.github) into [clientname]-[projectname]/.github
 
+Repo settings
+  - Disallow `Allow rebase merging` (because it is incompatible with `Require signed commits`)
```

Comment:
I guess this won't work after we active the required signed commits, so we could as well leave the default, right?

Reply:
It is confusing to people that they have the option, but it fail, when they click it.
A curious developer would then use time on searching for information.
If it is disabled, the curious developer would just ask someone in Abtion.